dockerfile/README.md

```diff
@@ -32,14 +31,25 @@ The Dockerfile defines the resources and instructions to build the Docker image
  * Access management console
      -  To access the management console, use the Docker host IP and port 9443.
          + `https://<DOCKER_HOST_IP>:9443/carbon`
```

Comment:
Shall we change DOCKER_HOST_IP to DOCKER_HOST ?
Please refer to https://github.com/wso2/docker-ei/tree/master/dockerfiles